### ISSUE TITLE
Unknown props message appears in log on 0.5.1

### ISSUE THREAD

**Issue by aalbul** [cheton/react-infinite-tree#6]
Hello,

After upgrade to 0.5.1 i realised that react prints this error in log:

Warning: Unknown props `autoOpen`, `loadNodes`, `noDataText`, `onSelectNode`, `rowRenderer`, `selectable`, `shouldSelectNode` on <div> tag. Remove these props from the element. For details, see https://fb.me/react-unknown-prop
    in div (created by _class2)
    in _class2 (created by TreeComponent)
    in TreeComponent (created by EntityTreeComponent)
    in EntityTreeComponent (created by EntityPanelComponent)
    in div (created by EntityPanelComponent)
    in EntityPanelComponent (created by Connect(EntityPanelComponent))
    in Connect(EntityPanelComponent) (created by ExplorerLayout)
    in section (created by ExplorerLayout)
    in div (created by Pane)
    in Pane (created by SplitPane)
    in div (created by SplitPane)
.....

Functionality works, but error is annoying.
This is how i define my tree:

```
import InfiniteTree from 'react-infinite-tree';

....
<InfiniteTree
                autoOpen={this.props.autoOpen}
                className={this.props.className}
                loadNodes={this.loadNodes}
                noDataText={this.props.noDataText}
                onDoubleClick={this.handleDoubleClick}
                onSelectNode={this.props.onNodeSelected}
                ref={this.assignTree}
                rowRenderer={this.rowRenderer}
                selectable={true}
                shouldSelectNode={this.isNodeSelected}
            />
```

So definition looks similar to one in examples. Is it a bug or i'm doing something wrong?

**Comment by cheton** [comment]
Ooops. That's my fault (https://github.com/cheton/react-infinite-tree/commit/7c0a83f2e91c227abcea95e89043f715a38baed5#diff-73642d587163f2eb4d72f0fa6ebfba02). I shouldn't pass all props to the container div, only "className" and "style" are necessary for the container. Thank you for reporting this issue, I will fix it soon.
```js
render() {
    return (
        <div {...this.props} />
    );
}
```

**Comment by cheton** [comment]
Fixed in v0.5.2
